### ISSUE TITLE
A11y_Visual Studio Code Services_Download_Platform Specific_Keyboard:Keyboard focus is not visible on image link.

### ISSUE THREAD

**Issue by poojapatil36** [microsoft/vscode-docs#5097]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Keyboard; #WCAG2.4.7; #FTP; #ChromiumEdge;

**Environment Details**:
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps**:
1. Launch the application Visual Studio Code Services
2. Navigate to "Download" button.
3. Hit on any of the platform(Windows/Linux/MacOS) links or any installer type(*64 and *32).
4. Observe that keyboard focus is not visible on image link present under visual studio code in service heading.

**Actual**:
Keyboard focus is not visible on image link present under visual studio code in service heading.

**Expected Result**:
keyboard focus should clearly visible on image link present under visual studio code in service heading

**User Impact**:
Users with low vision may not understand the content clearly if keyboard focus is not properly visible.

**Attachment**:
![A11y_Visual Studio Code Services_Download_Platform Specific_Keyboard](https://user-images.githubusercontent.com/97668775/149719894-60379cbe-b996-4fc9-90a8-fc5bc4e4fbe8.gif)

**Comment by gregvanl** [comment]
There is a focus outline around the image.

![image](https://user-images.githubusercontent.com/12818376/150621898-931e669f-d982-484b-911e-554fad275433.png)

**Comment by poojapatil36** [comment]
@gregvanl this bug is reproducing when we navigate using tab and the focus is not clearly visible on image link. The keyboard focus should create a solid outline border around the image link. So that user will able to see where the focus is. Please fix this issue.

**Comment by poojapatil36** [comment]
Verified the issue, issue is fixed.
Hence closing the bug. please find below attachment.
Github tag: #Closed;
![5097_A11y_Visual Studio Code Services_Download_Platform Specific_Keyboard](https://user-images.githubusercontent.com/97668775/218072351-16ffe9f1-cc93-489c-90da-4b94ed448294.png)
